src/index.ts

```diff
@@ -91,7 +99,26 @@ export async function waitForNotarize(opts: NotarizeWaitOptions): Promise<void>
     '-p',
     makeSecret(opts.appleIdPassword),
   ]);
-  if (result.code !== 0) {
+
+  // retry up to 10 times if getting 'Could not find the RequestUUID' error
+  // sometimes this is because service delay on apple service
+  while ((retry < 10) && (result.output.includes(RequestUUID_NOT_FOUND_ERROR_MSG))) {
+    d(`waiting for ${opts.initDelay} seconds before pinging Apple for status`);
+    await delay(opts.initDelay);
+    d(`retry #${retry}: pinging Apple for status with UUID`);
+    result = await spawn('xcrun', [
```

Comment:
We shouldn't duplicate the logic here, the first call should be removed and we should rely entirely on this while loop.  It might be easier to format this as a `do {} while ()` loop